robot_sort/robot_sort.py

```diff
@@ -96,8 +97,15 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+        
```

Comment:
Review different sorting methods like selection sort and bubble sort, is there a way you can implement them with these methods?